src/lib.rs

```diff
@@ -21,6 +20,8 @@
 use core::alloc::{Alloc, AllocErr, Layout};
 use core::cmp;
 use core::ptr;
+#[cfg(any(target_os = "linux", target_os = "macos", target_arch = "wasm32"))]
+pub use sys::Platform;
```

Comment:
If this needs to be `pub`, then the types need to be:

```
struct Platform {
  _private: ()
}
```
To prevent accidental construction